azure-pipelines.yml

```diff
@@ -193,12 +199,10 @@ stages:
               python.version: '3.8'
               pandas.version: 'latest'
               GE_pytest_pip_opts: '--requirement requirements-dev.txt --constraint constraints-dev.txt'
-# NOTE - 20210621 - JPC - as of this time, snowflake connector does not install on python 3.9; we are not
-# adding integration tests on that platform yet
-#            Python39:
-#              python.version: '3.9'
-#              pandas.version: 'latest'
-#              GE_pytest_pip_opts: '--requirement requirements-dev.txt --constraint constraints-dev.txt'
+            Python39:
```

Comment:
This successfully ran, logged [here](https://dev.azure.com/great-expectations/great_expectations/_build/results?buildId=12220&view=logs&j=74c24151-0aa8-5b89-790a-eb0187ad0335) and shown below, however the above `Python39-PandasLatest` didn't run -- why not? 

![image](https://user-images.githubusercontent.com/3752732/127560382-0a9003a3-b9ee-4aa9-abc1-5da51a63b41f.png)